packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -36,6 +37,14 @@ const EmbedPreview = ( {
 } ) => {
 	const [ isCaptionSelected, setIsCaptionSelected ] = useState( false );
 
+	const wrapperStyle = styles[ 'embed-preview__wrapper' ];
+	const wrapperAlignStyle =
+		styles[ `embed-preview__wrapper--align-${ align }` ] ||
+		styles[ 'embed-preview__wrapper--align' ];
+	const sandboxAlignStyle =
+		styles[ `embed-preview__sandbox--align-${ align }` ] ||
+		styles[ 'embed-preview__sandbox--align' ];
```

Comment:
Same for `embed-preview__sandbox--align`

Reply:
This has been addressed in [this commit](a88818615e6db3514a494ed60a33fb902c412589).